services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids is not None:
```

Comment:
Can we try to refactor this place and all other places in PR.
`if genres_ids is not None:` do we really need this extra check: `is not None`

Reply:
Explicity is better than implicity?